go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/Masterminds/semver v1.4.2
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
-	github.com/gardener/gardener v1.0.1
+	github.com/gardener/gardener v1.0.1-0.20200213093126-7a6123b6ae21
```

Comment:
Can you update to the sha of the current g/g master?

Reply:
wasn't sure if I should do that, this change is dependent on a specific commit, which is beyond that release use by the branch. I could do that, but is there an advantage in doing so?

Reply:
No problem, it can stay like this.